roles/openshift_node_dnsmasq/tasks/main.yml

```diff
@@ -65,3 +47,24 @@
 # Relies on ansible in order to configure static config
 - include: ./no-network-manager.yml
   when: not network_manager_active | bool
+
+- name: Reconfigure the node for dnsmasq
```

Comment:
This doesn't make any sense.  Why remove from the template just to put in in yedit here?